org/ystia/experimental/consul/linux/ansible/playbooks/consul_connects_agent_to_server.yml

```diff
@@ -29,6 +29,12 @@
       with_items: "{{ hostvars[inventory_hostname]  }}"
       when: item | match(TARGET_NODE+'_.*_SERVER_IP')
 
+    - name: compute consul servers
```

Comment:
Here if both SERVER_IP_CAP and SERVER_IP are resolved the same address will be added twice, isn't it?
I would definitely prefer having SERVER_IP_CAP overwriting SERVER_IP somehow.